examples/falcon/README.md

```diff
@@ -0,0 +1,76 @@
+# Falcon Sample Application for Pybrake
+
+## About the application
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time.
+2. `/locations` - returns list of available locations.
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API
+
+1. Install the dependencies for the application
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+2. You must get both `project_id` & `project_key`.
+
+   To find your `project_id` and `project_key` from Airbrake account and
+   replace it in below code in your project's `main.py` file.
+
+    ```python
+    import falcon
+    from pybrake.middleware.falcon import init_app
+   
+    app = falcon.App()
+   
+    config = dict(
+        PYBRAKE = dict(
+            project_id=999999,                               # Insert your Project Id here
+            project_key='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',  # Insert your Project Key here
+            environment="test",
+            performance_stats=True,                          # False to disable APM
```

Comment:
I think we can omit this option. We want to promote APM instead of suppressing the adoption.